app/controllers/volunteer_ops_controller.rb

```diff
@@ -75,6 +92,31 @@ def build_map_markers(organisations)
     end
   end
 
+  # This needs to be a helper method for the do-it API
+  def collect_all_items
+    host = 'https://api.do-it.org'
+    href = "/v1/opportunities\?lat\=51.5978\&lng\=-0.3370\&miles\=0.5 "
+    doit_orgs = []
+    id = 1
+    while href
+      url = host + href
+      response = HTTParty.get(url)
+      if response.body && response.body != '[]'
+        resp_items = JSON.parse(response.body)['data']['items']
+        resp_items.each do |item|
+          org = OpenStruct.new(latitude: item['lat'], longitude: item['lng'], name: item['title'], id: id,
+                                description: item['description'], opportunity_id: item['id'],
+                                org_name: item['for_recruiter']['name'], org_link: item['for_recruiter']['slug'])
```

Comment:
Metrics/LineLength: Line is too long. [113/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)